src/Badges.sol

```diff
@@ -19,27 +19,9 @@ contract Badges is
   ERC165Upgradeable,
```

Comment:
Are these deps redundant as BadgesStorage already inherit it?

Reply:
yup. fixed